crowbot/frontend/static/frontend/script.js

```diff
@@ -1,5 +1,41 @@
 // This file is loaded and executed when the main Crowbot page is opened.
 
+class ListManager {
+
+    constructor(listID){  //f.eks question-queue
+        this.list = listID;
+    }
+
+    addTextToList(content, cssClasses){ //cssClasses er ei liste.
+        var li = $("<li/>").text(content).addClass(cssClasses.join(" ")); // lager liste-element med klasser.
+        this.list.append(li)
+    }
+
+    prettyDatetime(datetime) {
+        return "[" + datetime.substring(0,10) + " " + datetime.substring(11,16) + "]";
+    }
+
+    addListItemWithTimeAndUser(text, usertype, username, timestamp, cssClasses) {
+        var subtext = "";
+        if (username == "Crowbot") {
+            subtext = "Answer by " + username; //Vi gidder ikke ha med "bot" og tid når Crowbot svarer
+        }else{
+            subtext = "Answer by " + usertype + " " + username + " " + prettyDatetime(timestamp);
```

Comment:
Alternatively, it can be made a function separate from the class -- it doesn't really have that much to do with managing lists, and could be useful elsewhere.